src/main/java/com/sphy/hotelmanagementapplication/service/RoomService.java

```diff
@@ -82,9 +84,14 @@ public List<RoomDTO> saveRooms(List<RoomDTO> roomsDTO) throws Exception{
 
         roomsSaved.spliterator().forEachRemaining(rooms::add);
 
+        List<RoomDTO> roomDTOS = new ArrayList<>();
+        for (Room room:rooms){
+            roomDTOS.add(roomToRoomDTO.converter(room));
+        }
+
 		rooms.clear();
```

Comment:
no need to invoke clear() here, the garbage collector wiil remove everything from memory after this method is executed